addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15185,13 +15194,13 @@ msgstr ""
 #. Description of setting "Pictures -> Resolution" with label #169
 #: system/settings/settings.xml
 msgctxt "#36310"
-msgid "No info available yet."
+msgid "No information available yet."
 msgstr ""
 
 #. Description of settings category "Pictures -> Slideshow" with label #108
 #: system/settings/settings.xml
 msgctxt "#36311"
-msgid "Category containing settings for how picture slideshows are handled."
+msgid "Category containing settings for how the picture slideshow is handled."
```

Comment:
Yes, Pictures is correct as there is more than one usually, same for Movies, TV shows etc.

Slideshow however is only one at any time for configuration no matter how hard you look at it. We cannot configure multiple slideshows on these options, there is no default addon installed whereby if it did configure multiple slideshows, then and only then this would be correct imo.